src/dafny/Unicode/Utf8EncodingForm.dfy

```diff
@@ -13,7 +13,7 @@ include "UnicodeEncodingForm.dfy"
 
 /**
   * The Unicode encoding form that assigns each Unicode scalar value to an unsigned byte sequence of one to four bytes
-  * in length, as specified in Table 3-6 and Table 3-7.
+  * in length.
```

Comment:
What's the reason for removing this citation? Would it be appropriate to instead add more context?

```suggestion
  * in length, as specified in Table 3-6 and Table 3-7 of the Unicode Standard, Version 14.0.
```